app/helpers/application_helper.rb

```diff
@@ -213,4 +213,10 @@ def site_wide_announcement
       session[:announcement_dismissals]
     ).first
   end
+
+  def show_pro_upsell?(user)
+    return false unless feature_enabled?(:alaveteli_pro)
```

Comment:
As we're just dealing with booleans here are guard clauses necessary, could be written as but is it as readable?

```suggestion
    feature_enabled?(:alaveteli_pro) && (!user || user && !user.is_pro?)
```

Reply:
Yeah I couldn't decide between this, but happy to reduce it down. Fixed in c7860024c.